### ISSUE TITLE
Positions Lab: Should not require at least 2-3 shapes to be selected 

### ISSUE THREAD

**Issue by leeyh20** [PowerPointLabs/PowerPointLabs#1576]
Suggestion: If only one shape is selected, should it be expected to work like the built-in powerpoint "Align" button, so that users don't need to go to Format--> Align to align a single shape?

**Comment by junkiattan** [comment]
Do you want the default align operation if only one shape is selected? Is this included in the desired behavior for the button?

**Comment by leeyh20** [comment]
Yeah was wondering if we should put the default align operation inside the feature if only one shape is selected so that it feels complete.
